sims/verisim/verilator.mk

```diff
@@ -1,7 +1,17 @@
-# Build and install our own Verilator, to work around versionining issues.
-VERILATOR_VERSION=3.920
+#########################################################################################
+# verilator installation makefrag
+#########################################################################################
+
+#########################################################################################
+# verilator version, binary, and path
+#########################################################################################
+VERILATOR_VERSION=4.008
```

Comment:
Check for buy-in on this version. I'm personally OK with it, but maybe we want to track RC's version? (What is RC's version?)

Reply:
RC also uses 4.008. There are issues with multithreading support but this still keeps it running on a single thread. See https://github.com/freechipsproject/rocket-chip/pull/1762 for more details.

Reply:
Also side note: This should be addressed in the PR for makefile changes rather than here.